### DISCUSSION TITLE
show mouse when presenting jw.org

### DISCUSSION THREAD

**Discussion by stefano799** [sircharlo/meeting-media-manager#1184] [Ideas] [answered]
good morning . thanks for the great work.
is it possible to have the possibility to see the mouse pointer in the reproduced web page?

**Comment by mtdvlpr** [comment, marked as answer]
Hi @stefano799, this is one of the improvements we want to implement as discussed in #761. The fact this and other improvements are missing is the reason that it is currently a BETA feature.

**Comment by mtdvlpr** [comment]
Hi @stefano799, good news! We're working on this improvement. You can track it at #1389 :)
